tutorials/haskell/generics/tutorial.md

```diff
@@ -0,0 +1,584 @@
+---
+title: GHC Generics Explained
+published: 2017-02-01
+ghc: 8.0.1
+lts: 7.16
+language: haskell
+author: Mark Karpov
+author-name: Mark Karpov
+description: This tutorial will get you up to speed with GHC generics quickly.
+---
+
+This tutorial will get you up to speed with GHC generics quickly. It should
+be noted that generics is not something academic and useless, quite the
```

Comment:
The first tweak seem to break the thought unnecessarily, i.e. the second part after semicolon does not clarify anything stated in the first part, it just follows it. The second tweak hardly improves anything, IMO.